drools-lsp-server/src/test/java/org/drools/lsp/server/DroolsLspDocumentServiceTest.java

```diff
@@ -42,7 +42,7 @@ void testReadRuleName() {
     }
 
     @Test
-    void testFindLHSandRHS() {
+    void findLHSandRHS() {
```

Comment:
See my previous comment.
As in previous test, here I think the tested method is `getCompletionItems`, but the test name `findLHSandRHS` does not suggest it: what about something like `getCompletionItems_FindLHSAndRHS` (or whatever you think it better describe whit this test is different from the others) ?

Reply:
@gitgabrio Agreed. I'll follow the practice. Thanks!